README.md

```diff
@@ -1,27 +1,89 @@
+For best results, the analyzer should run on trace files that were generated in the same relative location.
+If the trace files fall out of date with your project, you may see different results.
+https://github.com/microsoft/typescript-analyze-trace/issues/29
+You can run `npx analyze-trace --help` to find out about other options including:
 
-For a sorted list of compilation hot-spots, run `npx analyze-trace traceDir`.
-Pass `--help` to learn more about configuration options.
-For best results, run the analyzer on a machine where the paths in the trace file(s) resolve correctly.
+Option                    | Default | Description
+--------------------------|---------|-------------------------------------------------------------------------------
+`--skipMillis [number]`   | `100`   | Suppress events that take less than the specified number of milliseocnds
+`--color [boolean]`       | `true`  | Color the output to make it easier to read.
+`--expandTypes [boolean]` | `true`  | Expand the names of types when printing them.
+`--json [boolean]`        | `false` | *Experimental and unstable*: Produce JSON output for programmatic consumption.
 
-For a simplified view of a types file (useful when investigating an individual trace), run `npx simplify-trace-types traceDir\types.json output_path`.
+For a simplified view of a `types.json` file (useful when investigating an individual trace), you can run `npx simplify-trace-types traceDir/types.json output.txt`.
 Note that the resulting file is for human consumption and should not be passed to the analyzer (i.e. don't clobber the original).
 
-To pretty-print individual types from a types file (faster than processing the entire file), run `npx print-trace-types traceDir\types.json id+`.
+### Interpreting Results
+
+The `analyze-trace` output will try to highlight the most expensive portions of a compilation that it was able to measure (a.k.a. "hot spots").
+Each hot spot may have a breakdown of other contributing hot spots.
+
+`analyze-trace` will also try to point out when multiple versions of the same npm package were loaded and type-checked.
+
+Output currently looks like the following:
+
+```
+Hot Spots
+├─ Check file /some/sample/project/node_modules/typescript/lib/lib.dom.d.ts (899ms)
+├─ Check file /some/sample/project/node_modules/@types/lodash/common/common.d.ts (530ms)
+│  └─ Compare types 50638 and 50640 (511ms)
+│     └─ Compare types 50643 and 50642 (511ms)
+│        └─ Compare types 50648 and 50644 (511ms)
+│           └─ Determine variance of type 50492 (511ms)
+│              └─ Compare types 50652 and 50651 (501ms)
+|                 └─ ...
+├─ Check file /some/sample/project/node_modules/@types/babel__traverse/index.d.ts (511ms)
+└─ Check file /some/sample/project/node_modules/@types/react/index.d.ts (507ms)
+```
+
+Each step here is annotated with check times (e.g. checking two types in lodash took over 500ms).
```

Comment:
What do people think of

```suggestion
Each step here is annotated with check times (e.g. checking two types in lodash took over 500ms).

A step `s` is only displayed when they are

* `s.time >= skipMillis` and
   * `s.time >= forceMillis` or
   * `s.time >= (s.parent.time * 0.6)`
```

Reply:
The reason the `0.6` isn't configurable is that I specifically didn't want to expose (and thereby lock in) that heuristic.  I'd rather give approximate guidance on how to use the two parameters.

`skipMillis` indicates the minimum step duration that might be interesting.  If you'd like to see more output (perhaps because you're testing on a very fast computer), you can reduce this value.  Alternatively, if the output is too cluttered, you can increase it.

`forceMillis` indicates the minimum step duration that is definitely interesting - regardless of whether or not the step took a meaningful proportion of its parent step's time.  For example, you might try decreasing it if you have a step that takes 10 seconds and only 6 of those seconds are explained by the default output.  Note that decreasing `forceMillis` below `skipMillis` _will have no effect_.